### ISSUE TITLE
Rebuild Components.py doesn't work anymore

### ISSUE THREAD

**Issue by weiweihuanghuang** [mekkablue/Glyphs-Scripts#90]
On Version 2.3b (844)
```
Failed to rebuild Aogonek
Expecting instance of GSGlyphsInfo as self, got one of NSKVONotifying_GSGlyph
```

**Comment by mekkablue** [comment]
I was thinking of abandoning the script, because I only needed it once. Do you want to take it into your repository?

**Comment by schriftgestalt** [comment]
That should be fixed in the latest version (2.3-845)

**Comment by weiweihuanghuang** [comment]
Sure, it's a great script — I don't use it often but it saved a lot of time when I had to.

**Comment by mekkablue** [comment]
So, I did get another request after all. Just fixed it for 2.3.
